mam/v2/api/api.c

```diff
@@ -52,6 +52,27 @@ static void mam_api_bundle_wrap(bundle_transactions_t *const bundle,
   bundle_reset_indexes(bundle);
 }
 
+static trits_t mam_api_bundle_unwrap(bundle_transactions_t const *const bundle,
```

Comment:
Hell of a conflict with 899# :(

Reply:
899 is a draft based on this. It shouldn't conflict when rebased. I hope.